### PR TITLE
Added mf2.py in testcosmo

### PR DESCRIPTION
**Pull request by adrianplenefisch**
Added mf2.py, which is a python script that will test the simulation as a replacement for mf2.pro. Also modified mf.sh to make it run the new script, and to change some output file names from cube300.00128 that currently cause problems.

### CODE REVIEW

File: testcosmo/mf2.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#find percent error from the test function
+logn_pe = np.max(np.abs((logn-test_logn)/test_logn)*100)
+
+if logn_pe<5:
+    print("Congrats! This simulation produces a mass function as predicted!")
+    
+else:
+    print("PROBLEM!  This mass function differs from predicted by more than 5%!")
+
+print("The largest error is %.2f%%." % (logn_pe))
```
Comment (trquinn) [inline testcosmo/mf2.py:46]: Add a final "new line" to mf2.py

File: testcosmo/mf.sh [hunk 2]
```diff
@@ -8,9 +8,9 @@ SODIR=XXX
...
 echo "SKIDDING outputs"
-$SKIDDIR/skid -tau 0.005  -std -O 0.3 -H 2.894405 -s 12 -m 16 -o cube300.00128.skid -stats <cube300.000128
+$SKIDDIR/skid -tau 0.005  -std -O 0.3 -H 2.894405 -s 12 -m 16 -o cube300.000128.skid -stats <cube300.000128
 echo "Moving to SO"
-$SODIR/so -i cube300.00128.skid.gtp  -o cube300.00128.so -all -std -grp -gtp -stat cube300.00128.skid.stat -m 16 -O 0.3 -L  -p 1 -u 3.672e18 300 < cube300.000128
+$SODIR/so -i cube300.000128.skid.gtp  -o cube300.000128.so -all -std -grp -gtp -stat cube300.000128.skid.stat -m 16 -O 0.3 -L  -p 1 -u 3.672e18 300 < cube300.000128
 echo "SO complete"
 echo "Moving to Mass Function comparison"
-idl idl.mf.batch
+python3 mf2.py cube300.000128.so.sovcirc
```
Comment (trquinn) [inline testcosmo/mf.sh:16]: This can be either python2 or 3.  Use "python" so that the user's preference is chosen.

**Comment by trquinn** [comment]
This also addresses issue #144

File: testcosmo/mf2.py (hunk 1 above)
Comment (adrianplenefisch) [reply testcosmo/mf2.py]: Updated

File: testcosmo/mf.sh (hunk 2 above)
Comment (adrianplenefisch) [reply testcosmo/mf.sh]: Updated

**Pull request merged**